src/main/blackbox/blackbox.c

```diff
@@ -1502,6 +1502,11 @@ static bool blackboxWriteSysinfo(void)
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_YAW_DEADBAND, "%d",           rcControlsConfig()->yaw_deadband);
 
         BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_GYRO_HARDWARE_LPF, "%d",      gyroConfig()->gyro_hardware_lpf);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_GYRO_DELT, "%d",              gyroConfig()->gyro_delt);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_GYRO_DELTSQR, "%d",           gyroConfig()->gyro_deltSqr);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_GYRO_BITSHIFT, "%d",          gyroConfig()->gyro_bitshift);
+        BLACKBOX_PRINT_HEADER_LINE(PARAM_NAME_GYRO_UI, "%d",                gyroConfig()->gyro_ui);
```

Comment:
Maybe store ui order and bw in signle parameter? This is quite advanced configuration, user may do some math ...